src/components/network/network-renderer.js

```diff
@@ -664,6 +667,6 @@ genotet.NetworkRenderer.prototype.drawPath_ = function(terminator, coords,
         id: 'terminator',
         d: line([coords[0], coords[coords.length - 1]])
       });
-    this.svgPath_.select('path').remove();
+    this.svgPath_.selectAll('path').remove();
```

Comment:
The function is now act as that, with a new side adding into the polygon with n sides, it will draw a new path with n + 1 sides. Finally, it might be lots of overlapped paths. I wrote this part similar to one tutorial. I think a reasonable solution is to remove the previous n-side polygon and append a new one each time.